spec/services/document_url_spec.rb

```diff
@@ -32,7 +32,7 @@
   describe "#secret_preview_url" do
     it "returns the URL" do
       url = DocumentUrl.new(document).secret_preview_url
-      expect(url).to eq("https://draft-origin.test.gov.uk/foo?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiIzMzMxMzEzMS0zMzY1LTQ4MzgtYjk2My0zNDM3MzYzNTMxNjYifQ.nsYQ81gx2DKs2XQanFQIZzgkq_Ofw4C3Jys9II2RFoQ")
+      expect(url).to eq("https://draft-origin.test.gov.uk/foo?token=eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiIzMzMxMzEzMS0zMzY1LTQ4MzgtYjk2My0zNDM3MzYzNTMxNjYifQ.fFE5ctaewPYJap8hmsRyJ87L6a8Co8t9bbD0wpVYSgs")
```

Comment:
I wonder if older tokens will work with the newer version of JWT?

Reply:
Looks like it does :-).

```
content-publisher$master rails c
Loading development environment (Rails 5.2.2)
irb(main):001:0> JWT.encode({ "sub" => "foo" }, Rails.application.secrets.jwt_auth_secret)
=> "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJmb28ifQ.sPcJlsABv5OzEspmnKanhECSAsdb8tIdqSD0jHKYoOw"
irb(main):002:0> foo = JWT.encode({ "sub" => "foo" }, Rails.application.secrets.jwt_auth_secret)
=> "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJmb28ifQ.sPcJlsABv5OzEspmnKanhECSAsdb8tIdqSD0jHKYoOw"
irb(main):004:0> JWT.decode(foo, Rails.application.secrets.jwt_auth_secret)
=> [{"sub"=>"foo"}, {"typ"=>"JWT", "alg"=>"HS256"}]
irb(main):005:0> 
```

```
content-publisher$master gco dependabot/bundler/gds-sso-14.0.0
Switched to branch 'dependabot/bundler/gds-sso-14.0.0'
Your branch is behind 'origin/dependabot/bundler/gds-sso-14.0.0' by 1 commit, and can be fast-forwarded.
  (use "git pull" to update your local branch)
content-publisher$dependabot/bundler/gds-sso-14.0.0 rails c
Loading development environment (Rails 5.2.2)
irb(main):001:0> foo = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJmb28ifQ.sPcJlsABv5OzEspmnKanhECSAsdb8tIdqSD0jHKYoOw"
=> "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJmb28ifQ.sPcJlsABv5OzEspmnKanhECSAsdb8tIdqSD0jHKYoOw"
irb(main):002:0> JWT.decode(foo, Rails.application.secrets.jwt_auth_secret)
=> [{"sub"=>"foo"}, {"typ"=>"JWT", "alg"=>"HS256"}]
irb(main):003:0> JWT.encode({ "sub" => "foo" }, Rails.application.secrets.jwt_auth_secret)
=> "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJmb28ifQ.ZcnAtCai12C__x00CzfQAX5vmoBGNw0MmvLP6jsoPKk"
irb(main):004:0> 
```